llvm/lib/Target/RISCV/MCTargetDesc/RISCVELFStreamer.cpp

```diff
@@ -44,14 +44,33 @@ RISCVTargetELFStreamer::RISCVTargetELFStreamer(MCStreamer &S,
   // its ParseInstruction may call setForceRelocs as well.
   if (STI.hasFeature(RISCV::FeatureRelax))
     static_cast<RISCVAsmBackend &>(MAB).setForceRelocs();
+
+  // Using RISCVISAInfo, construct ISAString from given features.
```

Comment:
Can we use `RISCVFeatures::parseFeatureBits` here?

Reply:
Thanks - I missed that function.